tests/test_load.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+from pyhf_benchmark.load import downlaod, open_local_file, delete_downloaded_file
```

Comment:
I would just to `import pyhf_benchmark.load as load` and no import all the rest without namespaces. Makes it clearer what is happening in the API.

Also `downlaod` is spelled wrong, so that should get fixed here:

https://github.com/pyhf/pyhf-benchmark/blob/2313cc557385c47749ee6c5eab257150b298b9ae/src/pyhf_benchmark/load.py#L8

I think more descriptive function names should be used anyway. "download" doesn't really tell a user what is happening.

Reply:
It seems that the code check will show a wild import error if I use `import pyhf_benchmark.load as load`. I will try it again.